File: GSI QA testing tool/UI/UI_110_LaborUnion.cs
using GSI_QA_testing_tool.Utilities;
using OpenQA.Selenium;
using System;
using System.Collections.Generic;
using System.Diagnostics;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GSI_QA_testing_tool.UI
{
    internal class UI_110_LaborUnion
    {
        public static void GoTo(IWebDriver driver)
        {

            if (Data._StopAt.Contains("Labor Union Member"))
            {
                CustomDialog customDialog = new CustomDialog("Make Selection", "Select pages you want to stop at, make changes and click ok before clicking the Next button");
                customDialog.ShowDialog();
            }




            if (Finder.FindIt(driver, "//label[@for='ctl00_Main_content_ucUILaborUnion_rblUnionMember_1']")) 
            {
                Debug.WriteLine("LaborUnion is On");
            }
            else
            {
                Debug.WriteLine("LaborUnion is Off");
                return;
            }



            Finder.ClickIt(driver, "//label[@for='ctl00_Main_content_ucUILaborUnion_rblUnionMember_1']");
            Finder.ClickIt(driver, "//input[@id='ctl00_Main_content_btnNext']");
        }
    }
}

File: GSI QA testing tool/UI/UI_110_LaborUnion.vb
Imports OpenQA.Selenium
Imports System
Imports System.Collections.Generic
Imports System.Diagnostics
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace GSI_QA_testing_tool.UI
    Friend Class UI_110_LaborUnion
        Public Shared Sub [GoTo](ByVal driver As IWebDriver, ByVal js As IJavaScriptExecutor)
            If Data._StopAt.Contains("Labor Union Member") Then
                Data.StopAtDialog()
            End If

            If Finder.FindItByCSS(driver, "#ctl00_Main_content_ucUILaborUnion_rblUnionMember_1") Then
                Debug.WriteLine("LaborUnion is On")
            Else
                Debug.WriteLine("LaborUnion is Off")
                Return
            End If

            Finder.JSClickIt(driver, "#ctl00_Main_content_ucUILaborUnion_rblUnionMember_1", js)
            Finder.JSClickIt(driver, "#ctl00_Main_content_btnNext", js)
        End Sub
    End Class
End Namespace
